app/helpers/application_helper.rb

```diff
@@ -41,16 +41,14 @@ def current_locale
   end
 
   def billable_entry_checkbox(entry, entry_type)
-    if entry.billed
-      "√"
-    else
-      tag(:input,
-          type: "checkbox",
-          name: "#{entry_type}_to_bill[]",
-          class: "bill_checkbox",
-          value: "#{entry.id}",
-          "data-project-id" => entry.project.id)
-    end
+    class_name = entry.billed ? "billed" : "unbilled"
```

Comment:
this is still used in the billables.js to count the amount of checkboxes checked and differentiate between billed and unbilled.